helsingborg-stad/app-mitt-helsingborg
# Perf/change update case request

## Explain the changes you’ve made
Removed the responsibility for the app to pass case statuses to the api. Now instead the api handles the setting of statuses based on values passed while updating a case. Therefore behaviour in the app needed to change to support this new api change.

## Explain why these changes are made

These changes completes the moving of responsibility for setting case statuses from the app to the api.

## Explain your solution

This includes an update to the way we send the PUT request to the api in order to update an case. The changes made is in relation to the functionality that handles the request passing (updateCase) in case actions and where this function is called within the app.

## How to test the changes?

Concrete example:
1. Checkout this branch
2. Make sure that the api has an open case for a user that you can use along with bankid.
3. Fire upp the simulator by running the command`yarn ios`
4. Login using Bankid
5. Start the open case (no coapplicant should be present)
6. Answer the questions in the case form
7. Sign the form
8. The case should have the status "active:signature:completed"

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
